packages/compass/src/app/index.js

```diff
@@ -251,21 +249,14 @@ var Application = View.extend({
       this.tourClosed();
     }
   },
-  showOptIn: function() {
-    if (process.env.HADRON_ISOLATED !== 'true') {
-      const NetworkOptInView = require('./network-optin');
-      const networkOptInView = new NetworkOptInView();
-      this.renderSubview(networkOptInView, this.queryByHook('optin-container'));
-    }
-  },
   showSecurity: function() {
     app.appRegistry.getAction('Security.Actions').show();
   },
   tourClosed: function() {
     app.preferences.unset('showFeatureTour');
     app.preferences.save();
     if (!app.preferences.showedNetworkOptIn) {
-      this.showOptIn();
+      ipc.ipcMain.broadcastFocused('window:show-network-optin');
```

Comment:
This code doesn't run in the main thread, and so currently when running the app locally it breaks instead of showing the settings

![image](https://user-images.githubusercontent.com/5036933/177759576-9576aac8-bdde-47bb-b54b-59492691f3e1.png)